exporter/util.go

```diff
@@ -245,3 +249,22 @@ func renderBody(logger log.Logger, body config.Body, tplValues url.Values) (meth
 	}
 	return
 }
+
+// Initializes and returns a ValueConverter object. nil if there aren't any conversions
+func initializeValueConverter(metric config.Metric) config.ValueConverterType {
+	var valueConverters config.ValueConverterType
+
+	//convert all keys to lowercase
```

Comment:
I'm not positive that lower-casing all the keys allows this to cover all cases. JSON is case sensitive, so there could be multiple JSON keys that resolve to the same lowercase key. I'm also not very familiar with this code, so it may not matter.

Reply:
I have to revisit the code a little. Perhaps not having lower case is fine.

Reply:
Based on my memory, it would be best to keep the lower case in place unless it is common to have JSON keys that have fields with same characters but different character case.